### PR TITLE
Deprecated Old Pluribus Network netvisor Modules

### PR DESCRIPTION
**Pull request by rajaspachipulusu17**
##### SUMMARY
Deprecated Old Pluribus Network netvisor Modules

##### COMPONENT NAME
Pluribus Network netvisor Modules

##### ADDITIONAL INFORMATION
Removed all old Pluribus Networks modules as there is no support to these modules in latest Netvisor

### CODE REVIEW

**Comment by Akasurde** [comment]
@rajaspachipulusu17 Could you please address the issues [mentioned here](https://github.com/ansible/ansible/pull/48791#issuecomment-439360077)?

File: lib/ansible/modules/network/netvisor/_pn_cluster.py
Comment (Akasurde) [inline lib/ansible/modules/network/netvisor/_pn_cluster.py]: According to [this](https://docs.ansible.com/ansible/2.5/reference_appendices/release_and_maintenance.html#deprecation-cycle), `removed_in` should be current version of Ansible + 4 feature releases. That is `2.12` in this case. If you change 2.8 to 2.12 the error reported by Ansibot will go away. Let me know if you need any help.

**Comment by rajaspachipulusu17** [comment]
@Akasurde All checks have passed. can you point me how to merge this code to public. So that my other PR can be merged after this. I have some more modules to push.

**Comment by Akasurde** [comment]
Need another pair of eyes to verify this deprecation. cc @gundalow

File: lib/ansible/modules/network/netvisor/_pn_cluster.py
Comment (gundalow) [inline lib/ansible/modules/network/netvisor/_pn_cluster.py]:
```suggestion
  alternative: Latest modules will be pushed in Ansible 2.9 instead.
```

**Review by gundalow — CHANGES_REQUESTED** [review]
1) It seems like you are deprecating in 2.8 (current devel), though the new modules will not be available til 2.9. Is that correct, can we get the new modules in 2.8?
2) Needs adding to `docs/docsite/rst/porting_guides/porting_guide_2.8.rst`

File: lib/ansible/modules/network/netvisor/_pn_cluster.py
Comment (rajaspachipulusu17) [reply lib/ansible/modules/network/netvisor/_pn_cluster.py]: @gundalow @Akasurde  i have made the required changes. :)

Comment (rajaspachipulusu17) [reply lib/ansible/modules/network/netvisor/_pn_cluster.py]: Can you please let me know what needs to be added in `porting_guide_2.8.rst`?

Comment (Akasurde) [reply lib/ansible/modules/network/netvisor/_pn_cluster.py]: You need to add a note here - https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/porting_guides/porting_guide_2.8.rst#deprecation-notices

Comment (rajaspachipulusu17) [reply lib/ansible/modules/network/netvisor/_pn_cluster.py]: @Akasurde  I have added it. I am getting `This branch has conflicts that must be resolved` hope that will be taken care .

Comment (acozine) [reply lib/ansible/modules/network/netvisor/_pn_cluster.py]: @rajaspachipulusu17 you need to rebase the branch - see http://docs.testing.ansible.com/ansible/latest/dev_guide/developing_rebasing.html

**Comment by rajaspachipulusu17** [comment]
@acozine I have rebased and pushed it can you check now? and let me know if i want to change anything.

**Comment by rajaspachipulusu17** [comment]
Can i know status of this PR. Its been quite sometime in review. Can you please let me know what other steps should be done for merging this code. So that i can push my new modules.

**Comment by rajaspachipulusu17** [comment]
Will push the latest modules and then initiate to deprecate the old modules. Hence closing the PR.

**Comment by acozine** [comment]
@rajaspachipulusu17 sorry for the slow responses - it was a holiday in the US last week, and we are still digging through email and PR backlogs. When your new modules are ready, ping me and I'll look again at the deprecation changes.

**Comment by rajaspachipulusu17** [comment]
Hi,Thanks for the response. I will let you know.Can you let me know who will merge PR #49176? I don't have any issues with that PR and @amitsi has approved to merge it. Thanks,Rajas On 27-Nov-2018 8:23 PM, Alicia Cozine <notifications@github.com> wrote:@rajaspachipulusu17 sorry for the slow responses - it was a holiday in the US last week, and we are still digging through email and PR backlogs. When your new modules are ready, ping me and I'll look again at the deprecation changes.

**Comment by rajaspachipulusu17** [comment]
@acozine  can we what should i do to remove these modules. i am reopening this PR as i want to deprecated these modules.

Thanks,
Rajas

**Pull request closed without merging**
